app/src/androidTest/java/com/tpp/theperiodpurse/NavigationOnboardTest.kt

```diff
@@ -0,0 +1,162 @@
+package com.tpp.theperiodpurse
+
+import android.app.Application
+import androidx.activity.ComponentActivity
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.navigation.compose.ComposeNavigator
+import androidx.navigation.testing.TestNavHostController
+import com.tpp.theperiodpurse.ui.onboarding.OnboardViewModel
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import com.tpp.theperiodpurse.MainActivity
+import java.text.DateFormat
+import java.util.*
+import javax.inject.Inject
+
+@HiltAndroidTest
+class NavigationOnboardTest {
```

Comment:
Aside from testing the navigation, can you also test the values in the onboarding viewmodel? It's important to know if they actually record the values.